Study/task1.rb

```diff
@@ -0,0 +1,129 @@
+class Station
+
+  def initialize(name)
+    @name= name
+    @st = Array.new
+  end
+  
+  def take_train(train)
+    @st.push(train)
+  end
+  
+  def delete_train(train)
+    @st.delete(train)
+   
+    
+  end
+  
+  def list
+    puts "#{@st}"
+  end
+  
+  def count(type)
+    count_pass = 0
+    count_gruz = 0
+     #i = @st.count{|train| train == "pass"}
+     @st.each do |x|
+     if x == "pass"
+     count_pass += 1
+     elsif x == "gruz"
+     count_gruz += 1
+    end
+    end
+   if type == "pass"
+   return count_pass 
+    else return count_gruz 
+   end
+ 
+end
+end
+
+
+class Route
+  
+  def initialize(first_st, last_st)

```

Comment:
лучше использовать полные имена
```suggestion
  def initialize(first_station, last_station)
```
и далее в таком же стиле